lua/octo/ui/writers.lua

```diff
@@ -11,6 +11,21 @@ local vim = vim
 
 local M = {}
 
+-- Track if we've already warned about ProjectV2 config
+local projects_v2_config_warned = false
+
+--- Show a one-time warning about enabling ProjectsV2 config
+local function warn_projects_v2_config()
+  if not projects_v2_config_warned and not config.values.default_to_projects_v2 then
+    projects_v2_config_warned = true
+    vim.notify(
+      "ProjectsV2 timeline events are disabled. Enable them by setting 'default_to_projects_v2 = true' in your Octo config.",
+      vim.log.levels.WARN,
+      { title = "Octo.nvim" }
+    )
```

Comment:
Can you use the notify commands built out in `utils.lua` / `notify.lua`. This will break pre-commit

Reply:
Fixed